code/modules/events/wizard/greentext.dm

```diff
@@ -75,21 +75,21 @@
 /obj/item/greentext/process()
 	if(last_holder && last_holder != new_holder) //Somehow it was swiped without ever getting dropped
 		to_chat(last_holder, span_warning("A sudden wave of failure washes over you..."))
-		last_holder.add_atom_colour("#FF0000", ADMIN_COLOUR_PRIORITY)
+		last_holder.add_atom_colour("#ff0000", ADMIN_COLOUR_PRIORITY)
 		last_holder = new_holder //long live the king
 
 /obj/item/greentext/Destroy(force)
 	LAZYREMOVE(SSticker.round_end_events, roundend_callback)
 	QDEL_NULL(roundend_callback) //This ought to free the callback datum, and prevent us from harddeling
```

Comment:
oh btw this shouldn't be in `Destroy()`, because we want that to go by ASAP and a `for()` loop that handles cleanup not related to reference handling is quite poor practice. Would you mind moving everything from lines 84-92 into a new proc on `/obj/item/greentext` and then doing an `INVOKE_ASYNC()` so we don't clog up the `Destroy()` chain doing work that should be done elsewhere? If it's not critical to cleaning up the object before we delete it, it's best to ASYNC it out. I know it's not something you really touched here but I think it's pretty important to be done since this can really make the server weep for a little bit.